tutorials/general/t8_tutorial_search.cxx

```diff
@@ -171,10 +169,10 @@ t8_tutorial_search_callback (t8_forest_t forest, const t8_locidx_t ltreeid, cons
  * a counter for this element by one.
  * These counters are provided in an sc_array as user data of the input forest.
  */
-static int
+static void
 t8_tutorial_search_query_callback (t8_forest_t forest, const t8_locidx_t ltreeid, const t8_element_t *element,
                                    const int is_leaf, const t8_element_array_t *leaf_elements,
-                                   const t8_locidx_t tree_leaf_index, void *query, sc_array_t *query_indices,
+                                   const t8_locidx_t tree_leaf_index, sc_array_t *query, sc_array_t *query_indices,
```

Comment:
```suggestion
                                   const t8_locidx_t tree_leaf_index, sc_array_t *queries, sc_array_t *query_indices,
```

Reply:
Needs to be changed in the body of the function as well, but github doesnt let me do that here in review mode.